addons/website_forum/models/forum.py

```diff
@@ -193,7 +195,7 @@ def _get_plain_content(self):
 
     content_link = fields.Char('URL', help="URL of Link Articles")
     tag_ids = fields.Many2many('forum.tag', 'forum_tag_rel', 'forum_id', 'forum_tag_id', string='Tags')
-    state = fields.Selection([('active', 'Active'), ('close', 'Close'), ('offensive', 'Offensive')], string='Status', default='active')
+    state = fields.Selection([('active', 'Active'), ('pending', 'Waiting Validation'), ('close', 'Close'), ('offensive', 'Offensive'), ('flagged', 'Flagged')], string='Status', default='active')
```

Comment:
What I do is just a little bit different. I distinguish "close" and "offensive". They have respectively different reasons to "close"/"hide" the post (for questions and answers).
Someone flags: "flagged" (many users can flag a same post, and a "flags_counter" is incremented).
A moderator can then mark the post as "offensive" (with an "offensive reason" (racism, insults, ...)), or "active".